app/Filament/Resources/HubResource/RelationManagers/GamesRelationManager.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\HubResource\RelationManagers;
+
+use App\Filament\Resources\GameResource;
+use App\Filament\Resources\SystemResource;
+use App\Models\Game;
+use App\Models\GameSet;
+use App\Models\System;
+use App\Platform\Actions\AttachGamesToGameSetAction;
+use App\Platform\Actions\DetachGamesFromGameSetAction;
+use Filament\Forms;
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+class GamesRelationManager extends RelationManager
+{
+    protected static string $relationship = 'games';
+    protected static ?string $title = 'Games';
+    protected static ?string $icon = 'fas-gamepad';
+
+    public static function getBadge(Model $ownerRecord, string $pageClass): ?string
+    {
+        $count = $ownerRecord->games->count();
+
+        return $count > 0 ? "{$count}" : null;
+    }
+
+    public function table(Table $table): Table
+    {
+        return $table
+            ->modifyQueryUsing(fn (Builder $query) => $query->with('system'))
+            ->columns([
+                Tables\Columns\ImageColumn::make('badge_url')
+                    ->label('')
+                    ->size(config('media.icon.sm.width'))
+                    ->url(function (Game $record) {
+                        if (request()->user()->can('manage', Game::class)) {
+                            return GameResource::getUrl('view', ['record' => $record]);
+                        }
+                    }),
+
+                Tables\Columns\TextColumn::make('ID')
+                    ->label('ID')
+                    ->sortable()
```

Comment:
The default sort appears to be on ID. The user-facing page sorts on Game Title.

Also, sorting on Game Title here prioritizes the hacks.

Reply:
The title column is now the default sort, and the query under the hood is using `sort_title`.